data/TeamData.js

```diff
@@ -817,64 +829,81 @@ const MEMBERS = [
           ],
         ],
       },
-      // {
-      //   sectionName: "First Year Representatives",
-      //   members: [
-      //     [
-      //       {
-      //         image: Name_Image,
-      //         description: ``,
-      //         name: "First Last",
-      //         role: "",
-      //       },
-      //       {
-      //         image: Name_Image,
-      //         description: ``,
-      //         name: "First Last",
-      //         role: "",
-      //       },
-      //     ],
-      //   ],
-      // },
+      {
+        sectionName: "First Year Representatives",
```

Comment:
```suggestion
        sectionName: "First Year Representative",
```
Probably should update now that Mariam's changed roles